.github/workflows/lint.yml

```diff
@@ -31,3 +31,15 @@ jobs:
       - name: Running lint_whitespace_checks.sh on source files
         run: |
           ./build_tools/github_actions/lint_whitespace_checks.sh
+
+  llvm-integrate:
```

Comment:
Let's come up with a different name for this. I don't think that `llvm-integrate` captures what this code is doing particularly well.